oak_functions_launcher/benches/integration_benches.rs

```diff
@@ -41,18 +47,45 @@ struct OakFunctionsTestConfig {
 /// Similar to the integration test, but wrapped in a non-async function, and invoking the Wasm
 /// module in the benchmark loop.
 fn run_bench(b: &mut Bencher, config: &OakFunctionsTestConfig) {
+    env_logger::init();
     let runtime = tokio::runtime::Builder::new_multi_thread()
         .enable_all()
         .build()
         .unwrap();
 
+    runtime.block_on(xtask::testing::run_step(xtask::launcher::build_stage0()));
+    runtime.block_on(xtask::testing::run_step(xtask::launcher::build_binary(
+        "build Oak Restricted Kernel binary",
+        xtask::launcher::OAK_RESTRICTED_KERNEL_BIN_DIR
+            .to_str()
+            .unwrap(),
+    )));
+
     let oak_functions_linux_fd_bin_path =
```

Comment:
given that this is now enclave app, we should rename this var.

Reply:
Good catch, done.